test/emailVerificationTest.js

```diff
@@ -0,0 +1,87 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const { expect } = require('chai');
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const sendVerificationMail = require('../lib/utils/emailService/emailVerification');
```

Comment:
`sendVerificationMail` and `verifyNewUser ` should be tested in separate files

Reply:
> `sendVerificationMail` and `verifyNewUser ` should be tested in separate files

I will separate them immediately